### ISSUE TITLE
Upside down printing is not working.

### ISSUE THREAD

**Issue by constellates** [adafruit/Adafruit-Thermal-Printer-Library#19]
I'm trying to use this upside down feature and it doesn't change how text is printed at all.

```
  printer.upsideDownOn();
  printer.println(F("Test"));
  printer.upsideDownOff();
```

**Comment by constellates** [comment]
I added pull request #22 to fix this issue.

**Comment by sharkwheels** [comment]
I am also having this issue. Any fix for it yet?

**Comment by carolinebuttet** [comment]
Anyone?

**Comment by caternuson** [comment]
Fixed by #43
